### PR TITLE
Implement initial Activity, ViewModel

### PR DESCRIPTION
**Pull request by tbtzpdlql**
## The Issue
- [Implement "BaseActivity", "BaseViewModel"](https://github.com/the-boyj/wiki/issues/1) :  440a222
- [Add "MainActivity"](https://github.com/the-boyj/wiki/issues/2) : d500498

## Feature
- This PR just initial setting based on MVVM. So not any function.
- `BaseActivity` has `ViewDataBinding` and `BaseViewModel`. When create Activity, Binding each other.
- `BaseViewModel` has `CompositeDisposable`. It will be used by Rx.
- Add `Logger.class`
`Logger` shows Log as shown below. This can track where it was called. Also if clicks it, go to called location.
~~~java
Logger.d("Hello World");

D/BOYJ# MainViewModel.onCreate(MainViewModel.java:15): Hello World
~~~

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,5 +1,6 @@
+#Sat Jan 19 00:05:04 KST 2019
```
[the remark this answers is not in the document]
Comment (tbtzpdlql) [reply gradle/wrapper/gradle-wrapper.properties]: I updated Android studio - 3.3.0. So it was auto generated.
I'm sorry, I didn't tell.

File: app/src/main/java/com/webrtc/boyj/utils/Logg.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    public static void w(String msg) {
+        Log.w(tag(), msg);
+    }
+
+    public static void e(String msg) {
+        Log.e(tag(), msg);
+    }
+
+    private static String tag() {
+        int level = 4;
```
[the remark this answers is not in the document]
Comment (shooes270) [reply app/src/main/java/com/webrtc/boyj/utils/Logg.java]: No, I said 'these variables', all of the only one assigned variables.

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
[the remark this answers is not in the document]
Comment (shooes270) [reply gradle/wrapper/gradle-wrapper.properties]: My comment means that
'Is ```#Sat Jan 19 00:05:04 KST 2019``` auto generated by IDE?' not this file.

File: app/src/main/java/com/webrtc/boyj/utils/Logg.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (shooes270) [reply app/src/main/java/com/webrtc/boyj/utils/Logg.java]: amend commit plz.

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
[the remark this answers is not in the document]
Comment (tbtzpdlql) [reply gradle/wrapper/gradle-wrapper.properties]: It is updated time that Android Studio.
In general, This file add repository.

File: app/src/main/java/com/webrtc/boyj/utils/Logg.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (tbtzpdlql) [reply app/src/main/java/com/webrtc/boyj/utils/Logg.java]: Fixed in https://github.com/the-boyj/WebRTC_Android/pull/4/commits/ca1d27263949064d11176651e40b0ccb0fe8ece7

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
[the remark this answers is not in the document]
Comment (tbtzpdlql) [reply gradle/wrapper/gradle-wrapper.properties]: Ok, I understand. I deleted it.
Fixed in https://github.com/the-boyj/WebRTC_Android/pull/4/commits/ca1d27263949064d11176651e40b0ccb0fe8ece7

**Pull request merged**